.github/workflows/leo-build-tag-publish-and-run-tests.yml

```diff
@@ -76,8 +77,10 @@ jobs:
           if ${{ github.ref_name == 'develop' }}; then
             echo 'test-context=dev-merge'                              >> $GITHUB_OUTPUT
             echo 'relates-to-chart-releases=leonardo-dev'              >> $GITHUB_OUTPUT
+            echo 'leo-test-command=testOnly -- -l ExcludeFromJenkins'  >> $GITHUB_OUTPUT
           else
             echo 'test-context=pr-test'                                >> $GITHUB_OUTPUT
+            echo 'leo-test-command=testOnly -- -l "ExcludeFromJenkins ScheduledTest"'  >> $GITHUB_OUTPUT
```

Comment:
shouldn't this be in the if branch and not the else?

Reply:
I don't think so, we want to exclude the scheduled test from the pr context right?

Reply:
I renamed the Tag to follow the Exclude example so hopefully it should feel clearer now